api.go

```diff
@@ -56,7 +56,7 @@ func Server() {
 			err := godotenv.Load(".env")
 			util.Throw(err)
 			supabaseKey := os.Getenv("SUPABASE_SERVICE_ROLE_KEY")
-
```

Comment:
please format with `go fmt` to avoid extraneous diffs

Reply:
I was not aware of go fmt. Definitely going to give it a use next time